src/cargo/core/dependency.rs

```diff
@@ -209,7 +209,11 @@ impl Dependency {
         &self.inner.req
     }
 
-    pub fn name(&self) -> InternedString {
+    pub fn name_in_toml(&self) -> InternedString {
```

Comment:
Would you mind adding a short docstring to `name_in_toml`, `package_name`, and `rename`?  Perhaps with a little hint on when to use which?  I suspect future me is going to forget and be confused.

Reply:
Sure thing!